types/dadata/index.d.ts

```diff
@@ -0,0 +1,253 @@
+            full: string;
+            short: string;
+        };
+        okpo: Nullable<string>;
+        okato: Nullable<string>;
+        oktmo: Nullable<string>;
+        okogu: Nullable<string>;
+        okfs: Nullable<string>;
+        okved: string;
+        okved_type: string;
+        okveds: Nullable<string[]>;
+        authorities: null;
+        documents: null;
+        licenses: null;
+        phones: null;
+        emails: null;
+        employee_count: Nullable<string>;
+        finance: Nullable<{
+            tax_system: Nullable<string>;
+            income: Nullable<string>;
+            expense: Nullable<string>;
+            debt: Nullable<string>;
+            penalty: Nullable<string>;
+            year: Nullable<string>;
+        }>;
+        opf: {
+            code: string;
+            type: string;
+            full: string;
+            short: string;
+        };
+        management?: Nullable<{
+            name: string;
+            post: string;
+            disqualified: Nullable<string>;
+        }>;
+        founders: Nullable<string[]>;
+        managers: Nullable<string[]>;
+        predecessors: Nullable<string[]>;
+        successors: Nullable<string[]>;
+        branch_count?: number;
+        branch_type?: PartyBranchType;
+        address: Suggestion<PartyAddress>;
+        state: {
+            actuality_date: number;
+            registration_date: number;
+            liquidation_date: Nullable<number>;
+            status: PartyStatus;
+            code: Nullable<string>;
+        };
+        source: null;
+        qc: null;
+    }
+
+    type BankType = 'BANK' | 'BANK_BRANCH' | 'NKO' | 'NKO_BRANCH' | 'RKC' | 'OTHER';
+
+    type BankStatus = 'ACTIVE' | 'LIQUIDATING' | 'LIQUIDATED';
+
+    interface DaDataBank {
+        bic: string;
+        swift: string;
+        inn: string;
+        kpp: string;
+        registration_number: string;
+        correspondent_account: string;
+        name: {
+            payment: string;
+            full: null;
+            short: string;
+        };
+        payment_city: string;
+        opf: {
+            type: BankType;
+            full: null;
+            short: null;
+        };
+        address: Suggestion<Address>;
+        state: {
+            actuality_date: number;
+            registration_date: number;
+            liquidation_date: Nullable<number>;
+            status: BankStatus;
+        };
+        okpo: null;
+        phone: number;
+        rkc: number;
+    }
+
+    type Gender = 'MALE' | 'FEMALE' | 'UNKNOWN';
+
+    interface Fio {
+        surname: Nullable<string>;
+        name: Nullable<string>;
+        patronymic: Nullable<string>;
+        gender: Gender;
+        qc: '0' | '1';
+        source: null;
+    }
+
+    interface Email {
+        local: Nullable<string>;
+        domain: Nullable<string>;
+        type: null;
+        qc: null;
+        source: null;
+    }
+}
+
+export = DaData;
+export as namespace DaData;
```

Comment:
```suggestion
```
IMO it's just old school (check the no bracket syntax btw) CJS module:
https://unpkg.com/browse/dadata@0.0.3/index.js

Reply:
ok, fixed

Reply:
@peterblazejewicz i added your suggested change to index.d.ts, then build was broke ((